### ISSUE TITLE
🐛[BUG][System) Zoned into sky background, unable to move

### ISSUE THREAD

**Issue by coding4saudade** [HorizonFFXI/HorizonXI-Issues#796]
**What does this bug affect?**
After zoning into the city, my character get stuck and cannot move.  Very similar to the issue where we were load to to a black screen
Issue happens with going into Mog house too
**Steps to Reproduce**
Happens randomly, but zoning from east sarutaburta caused the issue for me.

**Expected behavior**
To be able to zone into city without getting stuck

**Screenshots**
screenshot added

**Additional context**
The other people on screen were stuck too.  Using the !unstuck command allows us to get out, but it sends us to our homepoint. for me that is jeuno which a far away from windurst.

before I got this sky bug, my client would error out to the black screen with downloading data spinning but nothing happening.

**Comment by iklarazu** [comment]
This had happened to me a few mins before and may be partly related to bug #669 

I was in a party, took a chocobo from Windurst Woods (time was [22:57:44] UTC 10 Jan 2023 if you have logs for that,) DRG17 so underlevel if that has any impact. Zone into E.Saruta, but no bird. First instinct is to obviously turn around and run back into the city to get a new one, which was when I got a black screen.

> [22:57:44] Sariale : You can rent a chocobo, but because your level is so low, you won't be able to ride for very long. Also, equipment that extends your riding time will have no effect.
[22:57:44] Sariale : If you still want one, it will cost you 255 gil.
[22:58:00] === Area: East Sarutabaruta ===

Upon force closing the game client and later logging back in, I was in the sky.

![image](https://user-images.githubusercontent.com/65548510/211685215-b42b4d37-29ab-46cb-aa7e-ac909efc2b87.png)

I tried to /shutdown bug logging back in I was still in the sky, and then saw several other players (Kitten, Venks, Jabutupaki) get put into the exact same position a few mins after me. Unfortunately I didn't think at the time to take a picture of that, but from the minimap overlay, I was south of the map, maybe like L20 or something.

I did use !unstuck and it did work, it's just a degrading and depressing amount of time to wait. I get it, stops abuse of the function, but at the same time, feels like punishment to those that need to use it legitimately.
